components/taggroup/metadata/taggroup.yml

```diff
@@ -80,28 +83,28 @@ examples:
     markup: |
       <div class= "spectrum-TagGroup" role="list" aria-label="Tags">
         <div class="spectrum-Tag spectrum-Tag--sizeS spectrum-TagGroup-item spectrum-Tag--removable" tabindex="0" role="listitem">
-          <span class="spectrum-Tag-label">Tag 1</span>
-          <button class="spectrum-ClearButton spectrum-ClearButton--sizeS" aria-label="Remove tag 1" tabindex="-1">
+          <span class="spectrum-Tags-itemLabel">Tag 1</span>
+          <button type="reset" class="spectrum-ClearButton spectrum-ClearButton--sizeS spectrum-Tag-clearButton" aria-label="Remove tag 1" tabindex="-1">
```

Comment:
That came from the example markup in **tag** when I was aligning the markup. I'll make these all consistent! Yeah, I think reset doesn't really make sense in this context since it's not a form. I may remove from the tag docs as well.